CHANGELOG.md

```diff
@@ -3,6 +3,26 @@
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org).
 
+## Supported Release [5.3.0]
+### Summary
+Implements rubocop changes within the module, alongside other smaller changes.
+
+#### Added
+- ensure=>absent added to postgresql::server:role.
+- Support added for Fedora 27.
+- scram-sha-256 added as a valid ph_hba_rule auth method.
+- 9.6 settings inherited for later PgSQL versions on FreeBSD.
+- A require has been added for puppet.
+
+#### Changed
+- Changes made to avoid the useless loading of files by augeas.
+- Modulesync changes.
+- psql_path defaulted to postgresql::server::psql_path.
+- Rubocop changes have been made.
+
+#### Removed
+- Debian 9 support decremented.
```

Comment:
Not sure if 'decremented' is the right term here?

Maybe 'deprecated' ?

Reply:
Good point

Reply:
Changed